source/libNsdl/src/sn_grs.c

```diff
@@ -23,7 +23,7 @@
  */
 #include <string.h>
 #include <stdlib.h>
-
+#include <assert.h>
```

Comment:
Why not? Assert.h is part of C99 and we're already using it all over codebase without any problems so far.